lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -319,6 +319,7 @@ def self.get_values(json, category_type, override_price = false)
       'x1e.xlarge' => 64, 'x1e.2xlarge' => 64, 'x1e.4xlarge' => 64, 'x1e.8xlarge' => 64, 'x1e.16xlarge' => 64, 'x1e.32xlarge' => 64,
       'p2.xlarge' => 64, 'p2.8xlarge' => 64, 'p2.16xlarge' => 64,
       'p3.2xlarge' => 64, 'p3.8xlarge' => 64, 'p3.16xlarge' => 64,
+      'f1.2xlarge' => 64, 'f1.4xlarge' => 64, 'f1.16xlarge' => 64,
```

Comment:
did we not have f1 at all here?

Reply:
We did not. Not sure why.

Reply:
are we sure of the CPU `bit width` here?  remember the f1 is a FPGA, and as I understand it is technically a horizontal (could be 256b wide) cpu?